Tasks/SqlAzureDacpacDeployment/DeploySqlAzure.ps1

```diff
@@ -39,6 +39,30 @@ Import-Module "Microsoft.TeamFoundation.DistributedTask.Task.DevTestLabs"
 # Load all dependent files for execution
 Import-Module ./Utility.ps1 -Force
 
+function Get-SingleFile($files, $pattern)
+{
+    if ($files -is [system.array])
+    {
+        throw (Get-LocalizedString -Key "Found more than one file to deploy with search pattern {0}. There can be only one." -ArgumentList $pattern)
+    }
+    else
+    {
+        if (!$files)
+        {
+            throw (Get-LocalizedString -Key "No files were found to deploy with search pattern {0}" -ArgumentList $pattern)
+        }
+        return $files
+    }
+}
+
+Write-Host "DacpacFile= Find-Files -SearchPattern $DacpacFile"
+$DacpacFilePath = Find-Files -SearchPattern $DacpacFile
+Write-Host "packageFile= $DacpacFilePath"
+
+#Ensure that at most a single package (.dacpac) file is found
+$DacpacFilePath = Get-SingleFile $DacpacFilePath $DacpacFile
```

Comment:
use -parameterName syntax to specify arguments for which valaue is passed. Like Get-SingleFile -files DacpacFilePath  -pattern $DacpacFile